include/openPMD/IO/IOTask.hpp

```diff
@@ -704,9 +677,10 @@ struct OPENPMDAPI_EXPORT Parameter<Operation::DEREGISTER>
     Parameter(Parameter const &) : AbstractParameter()
```

Comment:
CodeQL: Looks like `Parameter` generally lacks the copy assignment operator (independent of this PR):
```C++
Parameter& operator=(const Parameter& other)
```

https://github.com/openPMD/openPMD-api/security/code-scanning/498